src/config.ts

```diff
@@ -104,6 +114,11 @@ export const DEFAULT_QUERY_CONFIG: QueryConfig = {
 
   maxBinsList: [5, 10, 20],
 
+  // TODO: Come back and implement correctly when designing sort enumeration.
```

Comment:
I don't see you add a TODO in the follow up issue yet.  Very likely you will forget it if you don't.